src/Api/Auth/Models/Public/Request/AssociationWithPermissionsRequestModel.cs

```diff
@@ -4,12 +4,13 @@ namespace Bit.Api.Auth.Models.Public.Request;
 
 public class AssociationWithPermissionsRequestModel : AssociationWithPermissionsBaseModel
 {
-    public CollectionAccessSelection ToSelectionReadOnly()
+    public CollectionAccessSelection ToCollectionAccessSelection()
     {
         return new CollectionAccessSelection
         {
             Id = Id.Value,
-            ReadOnly = ReadOnly.Value
+            ReadOnly = ReadOnly.Value,
+            HidePasswords = HidePasswords.Value
```

Comment:
This property is nullable and is no longer `[Required]` so we need to handle that:

```suggestion
            HidePasswords = HidePasswords.GetValueOrDefault()
```